Find a functionally equivalent code:android.widget.DatePicker.getDayOfMonth:COMMENT
Method Modifier: public      
Comment:/**
 * @return The selected day of month.
 */

@@ -1,3 +1,3 @@
 {
-    return mCurrentDate.get(Calendar.DAY_OF_MONTH);
+    return mDelegate.getDayOfMonth();
 }

